src/index.css

```diff
@@ -0,0 +1,21 @@
+.nav-container {
```

Comment:
Looks like my .gitignore to remove .css files isn't working. Can you remove this file, and fiddle with the .gitignore so we aren't checking these in since they are generated?